package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fetch-openapi",
-  "version": "0.1.1",
+  "version": "0.1.2",
```

Comment:
oh nvm.  I see what happened, it is in it's own commit.

/me used to 1 commit PRs!